catalog/USGS/NLCD_RELEASES/USGS_RCMAP_RELEASES_V5_COMPONENTS.jsonnet

```diff
@@ -0,0 +1,358 @@
+              "013693",
+              "013491",
+              "012f8a",
+              "012d85",
+              "012c82",
+              "01297a"
+            ],
+            bands: [
+              'rangeland_annual_herbaceous',
+            ],
+          },
+        },
+      },
+    ],
+    'eo:bands': [
+      {
+        name: 'rangeland_annual_herbaceous',
+        description: |||
+          Percent of the pixel covered by annual herbaceous vegetation.
+        |||,
+        'gee:units': '%',
+      },
+      {
+        name: 'rangeland_bare_ground',
+        description: |||
+          Percent of the pixel covered by bare ground.
+        |||,
+        'gee:units': '%',
+      },
+      {
+        name: 'rangeland_non_sagebrush_shrub',
+        description: |||
+          Percent of the pixel covered by non-sagebrush shrubs.
+        |||,
+        'gee:units': '%',
+      },
+      {
+        name: 'rangeland_herbaceous',
+        description: |||
+          Percent of the pixel covered by herbaceous vegetation.
+        |||,
+        'gee:units': '%',
+      },
+      {
+        name: 'rangeland_litter',
+        description: |||
+          Percent of the pixel covered by litter.
+        |||,
+        'gee:units': '%',
+      },
+      {
+        name: 'rangeland_sagebrush',
+        description: |||
+          Percent of the pixel covered by sagebrush (*Artemisia* spp).
+        |||,
+        'gee:units': '%',
+      },
+      {
+        name: 'rangeland_shrub',
+        description: |||
+          Percent of the pixel covered by shrub.
+        |||,
+        'gee:units': '%',
+      },
+      {
+        name: 'rangeland_perennial_herbaceous',
+        description: |||
+          Percent of the pixel covered by perennial herbaceous vegetation.
+        |||,
+        'gee:units': '%',
+      },
+      {
+        name: 'rangeland_tree',
+        description: |||
+          Percent of the pixel covered by tree.
+        |||,
+        'gee:units': '%',
+      },
+    ],
+    rangeland_annual_herbaceous: {
+      minimum: 0.0,
+      maximum: 100.0,
+      'gee:estimated_range': false,
+    },
+    rangeland_bare_ground: {
+      minimum: 0.0,
+      maximum: 100.0,
+      'gee:estimated_range': false,
+    },
+    rangeland_non_sagebrush_shrub: {
+      minimum: 0.0,
+      maximum: 100.0,
+      'gee:estimated_range': false,
+    },
+    rangeland_herbaceous: {
+      minimum: 0.0,
+      maximum: 100.0,
+      'gee:estimated_range': false,
+    },
+    rangeland_litter: {
+      minimum: 0.0,
+      maximum: 100.0,
+      'gee:estimated_range': false,
+    },
+    rangeland_sagebrush: {
+      minimum: 0.0,
+      maximum: 100.0,
+      'gee:estimated_range': false,
+    },
+    rangeland_shrub: {
+      minimum: 0.0,
+      maximum: 100.0,
+      'gee:estimated_range': false,
+    },
+    rangeland_perennial_herbaceous: {
+      minimum: 0.0,
+      maximum: 100.0,
+      'gee:estimated_range': false,
+    },
+    },
```

Comment:
This line is the cause of the jsonnet file not being parsed.  Please remove it.

The build and check process showed this error:

```
ERROR: /home/runner/work/earthengine-catalog/earthengine-catalog/catalog/BUILD:6:17: Compiling Jsonnet to JSON for jsonnet_to_json_USGS_NLCD_RELEASES_USGS_RCMAP_RELEASES_V5_COMPONENTS.jsonnet failed: (Exit 1): bash failed: error executing command /bin/bash -c ... (remaining 1 argument skipped)

Use --sandbox_debug to see verbose messages from the sandbox and retain the sandbox build root for debugging
catalog/USGS/NLCD_RELEASES/USGS_RCMAP_RELEASES_V5_COMPONENTS.jsonnet:357:4-5 Did not expect: ","

  },
```

Reply:
removed line 351-had an extra "}," in there